app/src/test/java/gov/va/vro/architecture/NamingConventionTest.java

```diff
@@ -95,15 +95,15 @@ public void serviceProviderMapperShouldBeSuffixed() {
         .check(classes);
   }
 
-  @Test
-  public void serviceClassesShouldBeInServicePackage() {
-    classes()
-        .that()
-        .haveSimpleNameContaining("Service")
-        .should()
-        .resideInAPackage("..service..")
-        .check(classes);
-  }
+  //  @Test
```

Comment:
This is failing because of the addition of service location data models. It picks out the word "service" I believe. @yoomlam any way to remedy this?

Reply:
Yup!
```
Class <gov.va.vro.model.ServiceLocation> does not reside in a package '..service..' in (ServiceLocation.java:0)
```
Pushed a fix.

Reply:
Thank you!